Nomic_Analysis/Data/nbconverted/clean_data.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
```

Comment:
Like with the README comment, consider adding a small description with the point of this notebook and why you are cleaning the data (e.g. wanting to reproduce their results, etc.).